WalletWasabi/Blockchain/Transactions/TransactionStore.cs

```diff
@@ -33,7 +33,7 @@ public async Task InitializeAsync(string workFolderPath, Network network, string
 				Transactions = new Dictionary<uint256, SmartTransaction>();
 				TransactionsLock = new object();
 
-				var fileName = Path.Combine(WorkFolderPath, "Transactions.dat");
+				var fileName = Path.Combine(WorkFolderPath, $"Transactions.dat");
```

Comment:
Just a question why the `$` here?

Reply:
Because we were playing with the name of the file, trying with `$"Transactions.v{Version}.dat"` and we forgot to remove it.

Reply:
Done.